### DISCUSSION TITLE
SPI based LoRa HAT

### DISCUSSION THREAD

**Discussion by tomjones1977** [nzymedefense/nzyme#502] [Questions] [answered]
I've got two LoRa HATs on loan that use SPI not UART to comminucate with the Pi.  As a result they don't play at all with the test python files to read/write the config data to the LoRa module.  They are still based on the SX127X modules, just the wrong interface type.  Is there a way to configure the Pi to allow the nzyme code to interface with a pseudo serial interface?

https://www.dragino.com/products/lora/item/106-lora-gps-hat.html

**Comment by lennartkoopmann** [comment, marked as answer]
nzmye currently only support the UART interface. It could technically extended to use a different communication implementation but that's currently not on my roadmap
